wait_for_workers.py

```diff
@@ -66,7 +67,7 @@ def wait_for_workers(
 
     print("wait_for_workers.py - initializing client...", end="")
     sys.stdout.flush()
-    initialize_dask_cuda(communication_type)
+    #initialize_dask_cuda(communication_type)
```

Comment:
We no longer need to initialize the comms anymore because we already did in run-dask-process.sh?

Reply:
And since you comment out the function `initialize_dask_cuda` should we remove it?

Reply:
With updates to dask-cuda we no longer need to explicitly specify those flags an initialize dask-cuda on the client. Dask-cudf will automatically detect what options are available and enable/disable ib, rdma etc. We just need a few additional environment variables on the scheduler and client which I have updated this method to do with the latest changes